CONTRIBUTING.md

```diff
@@ -41,9 +41,14 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests!
 
-1. Fork the repo and create your branch from `master`.
+If you need help with GIT or our workflow, please ask on [Gitter.im](https://gitter.im/netlify/NetlifyCMS). We want your contributions even if you're just learning GIT. Our maintainers are happy to help!
+
+Netlify CMS uses the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) + [Feature Branches](https://www.atlassian.com/git/tutorials/comparing-workflows#feature-branch-workflow). Additionally, PR's should be [rebased](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) on master when opened, and again before merging.
+
+1. Fork the repo
+2. create a feature branch from `master`. If you're working on an issue, prefix your branch name with {{issue_number}}_description, otherwise use xx_description.
```

Comment:
Just drop the branch naming guidelines entirely, no need for it atm. I realize this was the whole point of the PR lol, but the rest of your changes add real value, so I'm glad for it :)

Reply:
When I was new to open source, explicit guidelines were nice because I didn't have to wonder if maintainers thought my choices were stupid. I totally understand being flexible in what you accept, though, so maybe word it as a recommendation rather than a requirement?

Reply:
@erquhart meet in the middle and make it a recommendation or strike it?

Reply:
@dopry sorry, thought I responded to this days ago - yeah, a recommendation would be fine.